proxy/README.md

```diff
@@ -0,0 +1,12 @@
+# Keycloak Security Proxy
+
+The Keycloak Security Proxy docker image.
+
+## Usage
+
+This image is intended to be used with an existing `/tmp/conf/proxy.json` on the host machine. 
+See [the proxy documentation](https://keycloak.gitbooks.io/documentation/server_installation/topics/proxy.html) 
+for more details.
```

Comment:
It does run without a certificate. If SSL is enabled but the keystore isn't specified, a cert is auto generated. In any case, the keystore can be added to the `/tmp/conf` along with the `proxy.json`, and referred within the `proxy.json` as `/tmp/conf/ssl.jks`